ftw/simplelayout/profiles/default/portlets.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<portlets>
+    <assignment
+      manager="plone.rightcolumn"
+      category="content_type"
+      key="ftw.simplelayout.ContentPage"
```

Comment:
If I'd like to not have a portlet slot, how do I disable the portlet?

We could use a context assignment instead, what are the pros / cons?